lib/components/joystick/joystick_directional.dart

```diff
@@ -66,77 +66,59 @@ class JoystickDirectional {
     _tileSize = size / 2;
   }
 
-  void initialize(Vector2 _screenSize, JoystickController joystickController) {
-    this._screenSize = _screenSize;
+  void initialize(Vector2 screenSize, JoystickController joystickController) {
+    _screenSize = _screenSize;
```

Comment:
This is wrong, it sets the global field to itself instead of to the argument.

Reply:
oops my bad

Reply:
bad linter as well :p